concourse/steps/release.py

```diff
@@ -740,6 +740,69 @@ def apply(self):
         logger.info('successfully posted contents to slack')
 
 
+class MergeReleaseToDefaultStep(TransactionalStep):
```

Comment:
considering those heavyweight transactional-steps did not really work out well, please rather merge this into eh existing step